src/libsync/discoveryphase.h

```diff
@@ -34,6 +34,15 @@ class Account;
  * if the files are new, or changed.
  */
 
+struct SyncOptions {
+    /** Maximum size (in Bytes) a folder can have without asking for confirmation.
+     * -1 means infinite */
+    qint64 _newBigFolderSizeLimit = -1;
```

Comment:
Just saying in future it would be nice if all variables with time or bytes are called something like ...msec or ..bytes etc. Not important now